### ISSUE TITLE
Cant see uploaded signatures

### ISSUE THREAD

**Issue by GhayyurAbbas** [docusign/code-examples-csharp#38]
I am using c# docusign quick start. According to document when i use createSignature method and pass imagebase64, signature name, uri etc, it only saves signature name, further more when i trigger embedded document signature method from local host i cant see saved signatures in options

**Comment by GhayyurAbbas** [comment]
{"errorCode":"INVALID_REQUEST_PARAMETER","message":"The request contained at least one invalid parameter. Invalid imageType: signature_image"}

**Comment by GhayyurAbbas** [comment]
Any response on this guys?

**Comment by InbarGazit** [comment]
This is an issue with the SDK, not the code examples, but we're aware of this, and it's in the queue for fix.

**Comment by GhayyurAbbas** [comment]
Yes i figured that out after doing some tests on your code example, if there is anything we can help with do let us know, because we are using docusign and saving signatures is one of our targets, we need this functionality asap.
